audformat/core/attachment.py

```diff
@@ -59,6 +60,72 @@ def __init__(
         self.path = path
         r"""Attachment path"""
 
+    @property
+    def files(
+            self,
+    ) -> typing.List[str]:
+        r"""List all files part of the attachment.
```

Comment:
```suggestion
        r"""List all files part of the attachment.
```
```suggestion
        r"""List all files that are part of the attachment.
```

Reply:
Or maybe: "List all files from the attachment"?

Reply:
No, your first suggestion is fine.